pipeline_manager/src/main.rs

```diff
@@ -588,10 +747,17 @@ struct NewConfigResponse {
 }
 
 /// Create a new project configuration.
-///
-/// # HTTP errors
-///
-/// * `NOT_FOUND` - `project_id` does not exist in the database.
+#[utoipa::path(
+    request_body = NewConfigRequest,
+    responses(
+        (status = OK, description = "Configuration successfully created.", body = NewConfigResponse),
+        (status = NOT_FOUND
+            , description = "Specified `project_id` does not exist in the database."
+            , body = ErrorResponse
+            , example = json!(ErrorResponse::new("Unknown project id '42'"))),
+    ),
+    tag = "Config"
+)]
 #[post("/new_config")]
```

Comment:
`POST /configs/`

Reply:
So a create operation usually does not use a verb, i.e., not `/configs/new`?

Reply:
Yup. POST is the verb, which usually creates something.